sdk/core/ci.yml

```diff
@@ -0,0 +1,16 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+pr:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - sdk/core/
```

Comment:
when core changes all dependent projects should be rebuilt.

cc @weshaggard

Reply:
Agreed. @mitch we should point this to eng/data-plane.proj.

Reply:
Updated.